src/models/itemFilterData.ts

```diff
@@ -0,0 +1,70 @@
+import type {
+  ItemAggregation,
+  ItemAggregationOption,
+  option as optionType,
+} from "../types/filterTypes"
+
+export class ItemFilterData {
+  options: ItemAggregationOption[]
+  _field: string
```

Comment:
@charmingduchess If the _field attribute doesn't need to be accessed externally, I'd suggest only having one "field" attribute and using a helper function to preprocess it for assignment in the constructor.

You can look at the getYearFromResult() helper function used to initialize the yearPublished attribute in the SearchResultsBib model as an example.